.github/ISSUE_TEMPLATE/project-onboarding.md

```diff
@@ -31,21 +31,69 @@ Please track your progress by using "Quote reply" to create your own copy of thi
-- [ ] Transfer [your domain to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63). The "LF Stakeholder email" is <projects@cncf.io>.
+- [ ] Transfer [your domain(s) to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63). The "LF Stakeholder email" is <projects@cncf.io>.
 - [ ] Transfer any [trademark and logo assets to the Linux Foundation](https://github.com/cncf/foundation/tree/main/agreements).
-- [ ] Submit a [pull request](https://github.com/cncf/artwork) with your artwork.
-- [ ] Transfer website analytics to <projects@cncf.io>. CNCF staff can help.
+- [ ] Submit a [pull request](https://github.com/cncf/artwork) with your existing artwork.  
+_Note_: If you don't have artwork, CNCF can help design some.
+- [ ] Transfer website analytics to <projects@cncf.io>, if applicable. CNCF staff can help.
 
 ## Update and document
 
 - [ ] Ensure that [DCO](https://github.com/apps/dco) (preferred) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project.
 - [ ] Ensure that that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's `README` on GitHub.
-- [ ] Ensure LF footer is on your website and [guidelines](https://github.com/cncf/foundation/blob/main/website-guidelines.md) are followed (if your project doesn't have a dedicated website, please adopt those guidelines for the `README` file).
 - [ ] Create a maintainer list and add it to the [aggregated CNCF maintainer list](https://maintainers.cncf.io) via pull request.
 - [ ] Provide emails for the maintainers to get access to the maintainers mailing list and Service Desk. Email them to <project-onboarding@cncf.io>.
 - [ ] Start working on [written, open governance](https://contribute.cncf.io/maintainers/governance/).
 - [ ] Start on an [OpenSSF Best Practices Badge](https://www.bestpractices.dev/en).
 
+## Websites
+
+_Note_, not all of these are applicable to all projects
+
+- [ ] Ensure website is
+  - [ ] [Hosted in an open source repo](https://github.com/cncf/techdocs/blob/main/docs/repo-setup.md)
+  - [ ] Hosted in the same organization as the main project
+  - [ ] Setup with [DCO](https://github.com/apps/dco) or CLA (DCO recommended)
+[CNCF's IP policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy)
+requires all projects to use either CLA (Contributor License Agreements)
+or [DCO (Developer Certificate of Origin)](https://github.com/apps/dco).
+Unless there's a strong necessity to use CLA, we encourage projects to
+use DCO as it's easier to setup and use.
```

Comment:
This may duplicate some instructions above, particularly line 42 "Ensure that [DCO](https://github.com/apps/dco) (preferred) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project."

We may want to drop it from here (though we may want to provide some of this info above)